.github/workflows/check-outdated-dependencies.yml

```diff
@@ -16,18 +18,12 @@ jobs:
       uses: actions/setup-java@v1
       with:
         java-version: 13
-    - uses: actions/cache@v1
-      with:
-        path: ~/.gradle/caches
-        key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
-        restore-keys: |
-          ${{ runner.OS }}-gradle-${{ env.cache-name }}-
-          ${{ runner.OS }}-gradle-
-          ${{ runner.OS }}-
-    - uses: actions/cache@v1
-      name: Cache gradle wrapper
```

Comment:
Did the fetching from the gradle executable jar from the cache take longer than fetching from gradle? - I wanted to reduce the GET requests to the gradle server by caching the wrapper binaries with this task.

Reply:
As this check is now run weekly, there is no need to optimize anything.